README.md

```diff
@@ -14,6 +14,48 @@ mode.
 * leveldb
 
 
+## Installing
+
+You will need a bitcoin node running on the same machine. You can safely use 
+the testnet. You will need this in your bitcoin.conf:
+```
+#testnet=1 # uncomment this for testnet
+txindex=1
```

Comment:
Actually this isn't needed!

In fact, you can even use a pruned node, which is what my alice and bob calendars do.